### DISCUSSION TITLE
Part 2 Q2

### DISCUSSION THREAD

**Discussion by awc17002** [cooperrc/computational-mechanics#123] [General]
I'm trying to make the matrix for question 2 and I'm getting the error, "list indices must be integers or slices, not tuple". I'm not sure how to fix it. I've included my code below: 

```
#Part A) 

a = 35 #degrees
b = 40 #degrees
F = 1 #kN

A = np.array([[1, np.cos(a), 0],
             [0, -2*np.cos(b/2), 0]
             [0, np.sin(a), 1]]) 
B = np.array([0, F, 0])

GaussNaive(A,B)
```

**Comment by gmg17005** [comment]
I might be crazy but what happens if you convert your angles to radians? Other than that, my code looks pretty much the same. That error comes up when an argument to a function (like one of the np functions or even a user defined function) is a list that you're not iterating over. If the argument is an array, the returning value should be an array. Also, try re-starting your kernel, save everything first tho

**Reply by awc17002** [reply]
That wasn't it but thank you for the response. I forgot a comma.

**Reply by cooperrc** [reply]
Nice find, 

without the comma, you get this error:

```python
      7 A = np.array([[1, np.cos(a), 0],
----> 8              [0, -2*np.cos(b/2), 0]
      9              [0, np.sin(a), 1]]) 
     10 B = np.array([0, F, 0])

TypeError: list indices must be integers or slices, not tuple
```
